app/code/Magento/Customer/Controller/Adminhtml/Group/NewAction.php

```diff
@@ -1,58 +1,28 @@
 <?php
 /**
- *
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
 namespace Magento\Customer\Controller\Adminhtml\Group;
 
 use Magento\Framework\App\Action\HttpGetActionInterface as HttpGetActionInterface;
-use Magento\Customer\Controller\RegistryConstants;
 
+/**
+ * Class NewAction
+ *
+ * @package Magento\Customer\Controller\Adminhtml\Group
+ */
 class NewAction extends \Magento\Customer\Controller\Adminhtml\Group implements HttpGetActionInterface
 {
     /**
-     * Initialize current group and set it in the registry.
+     * New customer group.
      *
-     * @return int
-     */
-    protected function _initGroup()
```

Comment:
please keep this function and mark as deprecated

Reply:
done